mockito-kotlin/src/main/kotlin/org/mockito/kotlin/Mocking.kt

```diff
@@ -112,7 +63,7 @@ inline fun <reified T : Any> mock(
     @Incubating useConstructor: UseConstructor? = null,
     @Incubating outerInstance: Any? = null,
     @Incubating lenient: Boolean = false,
-    stubbing: KStubbing<T>.(T) -> Unit
+    stubbing: KStubbing<T>.(T) -> Unit = {}
```

Comment:
We use this value on line 84 and now we unconditionally create a KStubbing even if you didn't specify it. I am okay if we refactor this method to check for an empty value and then creates the KStubbing if it isn't. Can you adapt the PR to do thst?

Reply:
Yes! I can do that. I'll make `stubbing` nullable, and give it a null default value, and only create the KStubbing if it is non-null..? Thanks for considering my PR

Reply:
Yes that works for me. Thanks for the PR and looking forward to the fix 😄

Reply:
Unfortunately, I think you are correct in that this would be a breaking change. I am not happy about the duplication either, but I don't think it is worth breaking our users for, given it is only a small burden for us. Therefore, I think it is better if we leave it as-is, unfortunately. Thanks for spotting this though!